<launch>

 <!-- Set Parameter Sever -->
 <rosparam  file="$(find controller)/cfg/param_server.yaml" command="load"/>

 <!-- Joy node -->
  <node respawn="true" pkg="joy" type="joy_node" name="jetson_joy" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
  </node>
 
 <!-- Pilot Node -->
  <node pkg="controller" name="autonomous_controller" type="controller.py" output="screen"/>

  <!-- Perception Node -->
  <param name="scale_linear" value="0.25" type="double"/>
  <param name="scale_angular" value="1.0" type="double"/>
  <param name="axis_linear" value="3" type="int"/>
  <param name="axis_angular" value="0" type="int"/>

  <node pkg="controller" type="controller_node" name="jetson_teleop" output="screen"/>
</launch>
